src/test/scala/chiselTests/experimental/hierarchy/Examples.scala

```diff
@@ -223,6 +223,12 @@ object Examples {
     @public val xy = (x, y)
   }
   @instantiable
+  class HasTuple5() extends Module {
+    val wire = Wire(UInt(3.W))
+    val inst = Module(new AddOne)
+    @public val tup = (3, wire, "hi", inst, List(1, 2, 3))
```

Comment:
Shouldn't this be deprecated, looking up the `inst` because it is a `Module` type?

Reply:
It is deprecated, but I've added warning suppression to the `build.sc`. I wanted to check that I implemented the tuple Lookupables correctly such that they still support the deprecated API. Follow on work will delete all of the deprecated stuff.